File: 시나공/007 배열과 문자열/예제3.java
// JAVA에서 향상된 for문을 사용한 예제

public class Example { 
	public static void main(String[] args) { 
		int[] a = {90, 100, 80, 70, 60, 50, 30}; 
		int hap = 0; 
		float avg; 
		for (int i : a) // a길이(7)만큼 i에 저장하면서 반복
			hap = hap + i; 
		avg = (float)hap / a.length; // a.length=7
		System.out.printf("%d, %.2f", hap, avg); 
	} 
}

// i=90 hap=90
// i=100 hap=90+100
// i=80 hap=90+100+80
// ...
// i=30 hap=90+100+80+70+60+50+30=480
// avg=480/7=68.571...

// 결과
// 480, 68.57

File: 시나공/012 Python의 활용/예제3.py
""" while문 """

i, hap = 0, 0
while i < 5: # i<5일 동안 계속 반복
    i += 1 
    hap += i
print(hap)

# i=0 => i=1 hap=1
# i=1 => i=2 hap=1+2
# i=2 => i=3 hap=1+2+3
# i=3 => i=4 hap=1+2+3+4
# i=4 => i=5 hap=1+2+3+4+5=15

# 결과 15